workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/SpatialOSBehaviourLibraryTests.cs

```diff
@@ -85,7 +85,8 @@ public void TearDown()
         [Test]
         public void SpatialOSBehaviourLibrary_injects_Reader()
         {
-            // TODO - Needs ReaderWriterFactory implemented to pass!
+            // TODO - Needs ReaderWriterFactory implemented to pass properly!
```

Comment:
this looks like something that could easily be forgotten... would prefer to have a ticket for that. we should not have TODOs in the code that are not associated with a ticket.

Also not a big fan of `Assert.Pass()` as it throws exceptions and therefore the whole thing could end up in a weird state. Just comment out the lines below and run an `Assert.IsTrue(true)` or remove the code completely and add it to the ticket

Reply:
Done and done: https://improbableio.atlassian.net/browse/UTY-732